backend/src/index.ts

```diff
@@ -1,6 +1,17 @@
 import express = require('express')
+import { userRouter } from './routes/user/users.router'
+import { projectRouter } from './routes/project/project.router'
+import { milestoneRouter } from './routes/milestone/milestone.router'
+import { testRouter } from './routes/tests/tests.router'
+
 const app = express()
 const PORT = 4000
+
+app.use('/users', userRouter)
+app.use('/projects', projectRouter)
+app.use('/milestone', milestoneRouter)
```

Comment:
You can change to "/milestones" to match other routes

Reply:
Will do that